### PR TITLE
WIP: README

### PR DESCRIPTION
**Pull request by andytang824**
Updated old env file.
Stashed old README.
Created new README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,93 +1,67 @@
...
+
+This tool provides a method for tools within the pipeline to report back to the Gitea through the use of issues and pull request comments.
+
+## Setup
+
+### Requirements
+
+- python v3.7 or higher
+- pip
+
```
Comment (Yousif-CS) [inline README.md:13]: add Snyk CLI as a requirement. hyperlink this https://support.snyk.io/hc/en-us/articles/360003812538-Install-the-Snyk-CLI

File: README.md [hunk 2]
```diff
@@ -1,93 +1,67 @@
...
+GITEA_USERNAME=USERNAME-TO-ACCESS-GITEA-AS
+GITEA_HOST=localhost
+GITEA_PORT=3000
+GITEA_REPO=YOUR-CODE-REPO
+SONAR_URL=YOUR-SONAR-URL
+SONAR_LOGIN=YOUR-SONAR-USERNAME-OR-TOKEN
+SONAR_PASSWORD=YOUR-SONAR-PASSWORD-BLANK-IF-TOKEN
+SONAR_KEY=YOUR-SONAR-PROJECT-KEY
+S3_BUCKET_NAME=YOUR-S3-BUCKET-NAME
+SNYK_TOKEN=YOUR-SNYK-TOKEN
```
Comment (Yousif-CS) [inline README.md:35]: You need to add those in the .env.example file since I couldn't see them

**Review by Yousif-CS — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (andytang824) [reply README.md:41]: These are introduced in this branch

**Review by Yousif-CS — APPROVED** [review]
lgtm

**Review by Johnathan-Yan — APPROVED** [review]

**Pull request merged**